### DISCUSSION TITLE
No matching indices found: No indices match pattern "logstash-*"

### DISCUSSION THREAD

**Discussion by scarl3t0** [Security-Onion-Solutions/securityonion#5951] [Unsupported Versions]
Hi all,

Total newbie to SO here and I have an issue with my Logstash/Logstash-Beats index. I'm currently using Winlogbeats to ship Sysmon logs to Logstash to be viewed in Kibana. I configured it initially and the default Logstash index was created and worked perfectly, but then I updated my deployment to 2.3.80 and now I have the issue seen in the pictures attached. No fields seem to be being mapped in the index. I have tried deleting the indexes, re-indexing, and manually loading to no avail. I am using a distributed model for my installation.

Any advice on how to troubleshoot would be very much appreciated! Apologies in advance for any missing information or bonehead mistakes in my setup. :) Thank you so much for reading.

![so1](https://user-images.githubusercontent.com/84025129/137986146-538641ec-d3c1-4192-a8bd-eb998b756f14.png)
![so2](https://user-images.githubusercontent.com/84025129/137986151-4e98bdea-56a9-4c32-b366-b856a48a7646.png)

**Comment by dougburks** [comment]
Did you follow the Winlogbeat instructions in our documentation?
https://docs.securityonion.net/en/2.3/beats.html

**Comment by abesinger** [comment]
Try searching using the `*:so-beats*` pattern instead of `*:logstash-*`  I'm using winlogbeat according to the instructions that Doug mentioned above, and my event logs  get indexed to `*:so-beats-*`.  FWIW, I don't think it makes any difference, but I'm not collecting sysmon at this point, just event logs.
